### PR TITLE
Save attachments

### PR DESCRIPTION
**Pull request by drparcon**

### CODE REVIEW

File: src/components/Attachment.svelte [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+<!-- {#if !attachments.length}Files{/if} -->
+<!-- <input
+    type="file"
+    {name}
+    {accept}
+    on:input
+    on:change={previewFiles}
+    {multiple}
+    {disabled}
+  /> -->
```
Comment (grantyap) [inline src/components/Attachment.svelte:104]: Remove this commented code.

File: src/components/Attachment.svelte [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+</Label>
+<CustomInput
+  type="file"
+  {name}
+  {accept}
+  on:change={previewFiles}
+  {multiple}
+  {disabled}
+/>
+<!-- </label> -->
```
Comment (grantyap) [inline src/components/Attachment.svelte:117]: Remove commented code.

File: src/components/Attachment.svelte [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+  </ul>
+{/if}
+
+<style>
+  /* .is-image {
+    height: auto;
+    width: 100%;
+    background-size: 100% auto;
+    background-repeat: no-repeat;
+  } */
```
Comment (grantyap) [inline src/components/Attachment.svelte:136]: Remove commented code

**Review by grantyap — CHANGES_REQUESTED** [review]

**Review by drparcon — COMMENTED** [review]
Nice.

**Pull request merged**